sdk/spring/azure-identity-spring/CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 
 ### Other Changes
 
+## 1.14.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.5-2.6.3.
```

Comment:
https://github.com/Azure/azure-sdk-for-java/pull/27147

This PR is not merged, and will merge after 1.14 released.
So this content should appear in 1.15's changelog.
I suggest you to create this PR after 1.14 released.